pkg/kubelet/dockershim/docker_service.go

```diff
@@ -81,6 +85,10 @@ const (
 	// to kubelet behavior and system settings in addition to any API flags that may be introduced.
 )
 
+var (
+	linuxIDMappingRegexp = regexp.MustCompile("([aA-zZ]+):([0-9]+):([0-9]+)")
```

Comment:
It seems that the first field can be numbers too. We probably need to modify this. Or maybe just use `Split()` with `:` as separator?

From https://docs.docker.com/engine/security/userns-remap/#prerequisites:

> Each file contains three fields: the username or ID of the user, followed by a beginning UID or GID (which is treated as UID or GID 0 within the namespace) and a maximum number of UIDs or GIDs available to the user

Reply:
Additionally, a typical accepted regex for usernames in Linux is `"^[a-z][-a-z0-9_]*\$"`, so need to account for numbers in the first field anyways.